base/interactiveutil.jl

```diff
@@ -49,6 +49,8 @@ function edit(path::AbstractString, line::Integer=0)
         cmd = line != 0 ? `$command $path -l $line` : `$command $path`
     elseif startswith(name, "subl") || startswith(name, "atom")
         cmd = line != 0 ? `$command $path:$line` : `$command $path`
+    elseif startswith(name, "code")
```

Comment:
So check something like `name == "code" || name == "Code.exe"` where the second name seems to be the one of the executable on Windows?